src/main/java/racingcar/manager/GameManager.java

```diff
@@ -1,43 +1,45 @@
 package racingcar.manager;
 
 import racingcar.model.Car;
+import racingcar.ui.UIInGame;
 import racingcar.util.RandomNumberGenerator;
 
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 public class GameManager {
-    UIManager uiManager = new UIManager();
+    private UIManager uiManager = new UIManager();
+    private Map<Integer, Car> carList = new HashMap<>();
+
     public void startGame(){
-        introScene();
-    }
-    private void introScene() {
         int inputNum = uiManager.showIntro();
         if (inputNum == 1){
-            inGame(uiManager.showStartInGame());
+            Integer carCount = uiManager.showCarCount();
+            createCar(carCount);
+            runRound(carCount, uiManager.showRoundCount());
         }
         if (inputNum == 2){
             System.exit(0);
         }
     }
-    private void inGame(List<Integer> inputList) {
-        Integer carCount = inputList.get(0);
-        Map<Integer, Car> carList = new HashMap<>(carCount);
-        Integer roundCount = inputList.get(1);
-        RandomNumberGenerator randomNumberGenerator = new RandomNumberGenerator();
+
+    private void createCar(Integer carCount){
         for (int carNum = 1; carNum <= carCount; carNum++) {
             Car car = new Car(carNum, 0);
```

Comment:
```suggestion
            Car car = new Car(carNum);
```

위와 같이 구현하고 Car를 생성할 때 현재 위치 값을 0으로 초기화하는 방식으로 구현하는 것은 어떨까?